src/pybind/mgr/cephadm/module.py

```diff
@@ -91,7 +91,7 @@ def remoto_has_connection(self: Any) -> bool:
 """
 
 # Default container images -----------------------------------------------------
-DEFAULT_IMAGE = 'docker.io/ceph/ceph'
+DEFAULT_IMAGE = 'quay.io/ceph/ceph'
 DEFAULT_PROMETHEUS_IMAGE = 'docker.io/prom/prometheus:v2.18.1'
```

Comment:
I was wondering if wz need to switch from docker.io to quay.io registry for monitoring and ingress container images but only the alertmanager/node-exporter/prometheus images are available on quay.io.

```python
DEFAULT_PROMETHEUS_IMAGE = 'quay.io/prometheus/prometheus:v2.18.1'
DEFAULT_NODE_EXPORTER_IMAGE = 'quay.io/prometheus/node-exporter:v0.18.1'
DEFAULT_GRAFANA_IMAGE = 'docker.io/ceph/ceph-grafana:6.7.4'
DEFAULT_ALERT_MANAGER_IMAGE = 'quay.io/prometheus/alertmanager:v0.20.0'
DEFAULT_HAPROXY_IMAGE = 'docker.io/library/haproxy:2.3'
```

I think switching more container images to quay.io registry (in addition to the ceph container image) is a good thing for users to avoid rate limit but on the other hand we're mixing docker.io/quay.io for the monitoring/ingress part in the configuration which could be confusing.

What do you think ?